latest/ug/automode/auto-accelerated.adoc

```diff
@@ -0,0 +1,342 @@
+          operator: In
+          values:
+          - g6e
+          - g6
+      terminationGracePeriod: 24h0m0s
+----
+
+workload.yaml
+
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nvidia-smi
+spec:
+  nodeSelector:
+    eks.amazonaws.com/instance-gpu-memory: "45776"
+    eks.amazonaws.com/compute-type: auto
+  restartPolicy: OnFailure
+  containers:
+  - name: nvidia-smi
+    image: nvidia/cuda:12.8.0-cudnn-devel-ubuntu20.04
+    args:
+    - "nvidia-smi"
+    resources:
+      requests:
+        memory: "30Gi"
+        cpu: "3500m"
+        nvidia.com/gpu: 1
+      limits:
+        memory: "30Gi"
+        nvidia.com/gpu: 1
+----
+
+
+To remove all objects created, simply run the following commands:
+
+----
+kubectl delete -f .
+----
+
+== Example NodePools Reference
+
+=== Create an NVIDIA NodePool
+
+The following NodePool defines:
+
+* Only launch instances of `g6e` and `g6` family
+* Consolidate nodes when empty for 1 hour
+** The 1 hour value for `consolodateAfter` supports spiky workloads and reduce node churn. You can tune `consolidateAfter` based on your workload requirements.
+
+*Example NodePool with GPU instance family and consolidation*
+
+[source,yaml]
+----
+apiVersion: karpenter.sh/v1
+kind: NodePool
+metadata:
+  name: gpu
+spec:
+  disruption:
+    budgets:
+    - nodes: 10%
+    consolidateAfter: 1h
+    consolidationPolicy: WhenEmpty
+  template:
+    metadata: {}
+    spec:
+      nodeClassRef:
+        group: eks.amazonaws.com
+        kind: NodeClass
+        name: default
+      requirements:
+        - key: "karpenter.sh/capacity-type"
+          operator: In
+          values: ["on-demand"]
+        - key: "kubernetes.io/arch"
+          operator: In
+          values: ["amd64"]
+        - key: "eks.amazonaws.com/instance-family"
+          operator: In
+          values:
+          - g6e
+          - g6
+      terminationGracePeriod: 24h0m0s
+----
+
+In addition, GPU workloads might be sensitive to specific GPU memory requirements, you might set a `eks.amazonaws.com/instance-gpu-memory` node selector on workloads so they schedule on nodes that have the required GPU memory resources. For other well-known labels which influence scheduling review, see <<auto-supported-labels>>.
+
+If you have specific storage requirements you can tune the nodes ephemeral storage `iops`, `size` and `throughput` by creating your own xref:create-node-class[NodeClass] to reference in the NodePool. Learn more about the xref:create-node-class[configurable NodeClass options].
+
+*Example storage configuration for NodeClass*
+
+[source,yaml]
+----
+apiVersion: eks.amazonaws.com/v1
+kind: NodeClass
+metadata:
+  name: gpu
+spec:
+  ephemeralStorage:
+    iops: 3000
+    size: 80Gi
+    throughput: 125
+----
+
+=== Define an {aws} Trainium and {aws} Inferentia NodePool
+
+The following NodePool has an `eks.amazonaws.com/instance-family` set that says, only launch instances of Inferentia and Trainium family:
```

Comment:
Change `eks.amazonaws.com/instance-family` to `eks.amazonaws.com/instance-category`